includes/Core/Authentication/Authentication.php

```diff
@@ -653,8 +680,9 @@ private function get_rest_routes() {
 						'methods'             => WP_REST_Server::READABLE,
 						'callback'            => function( WP_REST_Request $request ) {
 							$data = array(
-								'connected'  => $this->credentials->has(),
-								'resettable' => $this->options->has( Credentials::OPTION ),
+								'connected'      => $this->credentials->has(),
+								'resettable'     => $this->options->has( Credentials::OPTION ),
+								'setupCompleted' => $this->is_setup_completed(),
```

Comment:
Typo here, the method name is `is_setup_complete()`. It's a bit inconsistent between JS and PHP, but let's keep the `setupCompleted` (past tense) for JS, as it is more in line with e.g. `connected`.

Or we rename the PHP method as well, to `is_setup_completed` - I think that'd be even better. We're moving it anyway, so no "worse" breaking implications from doing that.